drivers/matrix_keypad/matrix_keypad.c

```diff
@@ -91,7 +91,7 @@ int matrix_keypad_init(matrix_keypad_t *dev, const matrix_keypad_params_t *param
         (CONFIG_MATRIX_KEYPAD_DEBOUNCE_PATTERN_BEGIN & CONFIG_MATRIX_KEYPAD_DEBOUNCE_PATTERN_END) == 0,
         "Debounce patterns must not overlap");
     memset(dev, 0, sizeof(matrix_keypad_t));
```

Comment:
not sure but would 

```suggestion
    *dev = (matrix_keypad_t){0};
```
be easier to read ?

Reply:
I find the current to be easier to read, but that might be because I'm used to the pattern. Anyway, unrelated to this PR right, so if you insist could you open a new PR with the suggested change?